lerobot/scripts/eval.py

```diff
@@ -535,6 +545,8 @@ def main(
     else:
         # Note: We need the dataset stats to pass to the policy's normalization modules.
         policy = make_policy(hydra_cfg=hydra_cfg, dataset_stats=make_dataset(hydra_cfg).stats)
+
+    assert isinstance(policy, nn.Module)
```

Comment:
I have 2 comments earlier about this check. Leaving this note here to make sure we resolve.